### DISCUSSION TITLE
Documentation clarification

### DISCUSSION THREAD

**Discussion by majordoobie** [ghostty-org/ghostty#2898] [Feature Requests, Ideas] [answered]
Good evening team! I am still testing and taking notes, but wanted to see what the consequences were on these couple of notes I have taken so far. Wanted to know if it was worth making a PR for it? 

# Things that need clarification

## Add instructions for path

This will probably be fixed when a brew package is made, but it would be helpful 
to demostrate how to set ghostty into your path

```bash
# Install using instructions provided
sudo ln -s /Applications/Ghostty.app/Contents/MacOS/ghostty /opt/ghostty
export PATH="/opt:$PATH"
```

## cli actions vs keymap actions

There are actions that you can execute that are listed in `ghostty +help`. There is also 
a list of actions that you can print with `ghostty +list-actions`. The wording makes
it sound as if you can use the actions in `ghostty +list-actions` in the CLI like `reload_config`

However, that is not the case. The actions in `ghostty +list-actions` are for config file actions
like setting a keymap: 

```ini
keybind = super+shift+comma=reload_config
```

Additionally, there is a style missmatch with `list_actions` vs `reload_config` 

## Commands that require restart but not labled
title = 
window-decoration =

**Comment by mitchellh** [comment]
Will respond to each thing separately.

> Commands that require restart but not labled
> 
> title =
> window-decoration =

This should be fixed. 

1. `title` should not require a restart, so rather than fix the config, we should fix the implementation. We can do this at runtime.
2. `window-decoration` should not require a restart and should already apply to new windows only (many other configs are documented this way).

I'd accept PRs for either of those.

**Reply by mitchellh** [reply]
> title should not require a restart, so rather than fix the config, we should fix the implementation. We can do this at runtime.

Fixed in #2906

**Comment by mitchellh** [comment]
> cli actions vs keymap actions

I think we should rename the former to `cli commands` to avoid the confusion.

**Reply by majordoobie** [reply]
Just noticed I have a typo 

from:
Additionally, there is a style mismatch with `list_actions` vs `reload_config`

To:
Additionally, there is a style mismatch with `list-actions` vs `reload_config`

(not a bit deal if this is what it was meant to be. Just an observation)

**Comment by 00-kat** [comment, marked as answer]
Stumbled across this randomly:

> Add instructions for path

Not relevant anymore I'd think, since we have https://ghostty.org/download now.

> cli actions vs keymap actions

Discussion about this should probably be moved to a separate discussion (such as #5622, not sure if there are any other ones about this), or this discussion should be renamed, since it's *incredibly hard* to find this discussion when searching (GitHub's search being subpar does not help either).
